### ISSUE TITLE
Error installing v1.24 update

### ISSUE THREAD

**Issue by YourAverageCoderr** [Microsoft/vscode#52233]
After installing the update for v1.24 in VSCode, VSCode told me "Failed to install VS Code update. Please download and reinstall VS Code"

- VSCode Version: v1.23
- OS Version: Windows 10 build 17134

Log:
```
Jun 18 12:03:39.783 INFO Starting: C:\Program Files\Microsoft VS Code\Code.exe, true
Jun 18 12:03:39.903 INFO Checking for running Code.exe processes... (attempt 1)
Jun 18 12:03:39.918 INFO Code.exe is running, wait a bit
Jun 18 12:03:40.423 INFO Checking for running Code.exe processes... (attempt 2)
Jun 18 12:03:40.443 INFO Code.exe is running, wait a bit
Jun 18 12:03:40.948 INFO Checking for running Code.exe processes... (attempt 3)
Jun 18 12:03:40.968 INFO Code.exe is running, wait a bit
Jun 18 12:03:41.473 INFO Checking for running Code.exe processes... (attempt 4)
Jun 18 12:03:41.478 INFO Code.exe is running, wait a bit
Jun 18 12:03:41.983 INFO Checking for running Code.exe processes... (attempt 5)
Jun 18 12:03:41.993 INFO Code.exe is running, wait a bit
Jun 18 12:03:42.498 INFO Checking for running Code.exe processes... (attempt 6)
Jun 18 12:03:42.503 INFO Code.exe is running, wait a bit
Jun 18 12:03:43.009 INFO Checking for running Code.exe processes... (attempt 7)
Jun 18 12:03:43.014 INFO Code.exe is not running
Jun 18 12:03:43.014 INFO Checking for possible conflicting running processes... (attempt 1)
Jun 18 12:03:43.024 INFO Inno Updater v0.4.3
Jun 18 12:03:43.024 INFO Starting update, silent = true
Jun 18 12:03:43.259 INFO do_update: "C:\\Program Files\\Microsoft VS Code\\Code.exe", _
Jun 18 12:03:43.259 INFO move_update: "C:\\Program Files\\Microsoft VS Code\\unins000.dat", _
Jun 18 12:03:43.279 INFO Delete: "api-ms-win-core-console-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.309 INFO Delete: "api-ms-win-core-datetime-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.314 INFO Delete: "api-ms-win-core-debug-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.314 INFO Delete: "api-ms-win-core-errorhandling-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.314 INFO Delete: "api-ms-win-core-file-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.314 INFO Delete: "api-ms-win-core-file-l1-2-0.dll" (attempt 1)
Jun 18 12:03:43.314 INFO Delete: "api-ms-win-core-file-l2-1-0.dll" (attempt 1)
Jun 18 12:03:43.314 INFO Delete: "api-ms-win-core-handle-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.314 INFO Delete: "api-ms-win-core-heap-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.319 INFO Delete: "api-ms-win-core-interlocked-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.319 INFO Delete: "api-ms-win-core-libraryloader-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.319 INFO Delete: "api-ms-win-core-localization-l1-2-0.dll" (attempt 1)
Jun 18 12:03:43.319 INFO Delete: "api-ms-win-core-memory-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.319 INFO Delete: "api-ms-win-core-namedpipe-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.319 INFO Delete: "api-ms-win-core-processenvironment-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.324 INFO Delete: "api-ms-win-core-processthreads-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.324 INFO Delete: "api-ms-win-core-processthreads-l1-1-1.dll" (attempt 1)
Jun 18 12:03:43.324 INFO Delete: "api-ms-win-core-profile-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.324 INFO Delete: "api-ms-win-core-rtlsupport-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.324 INFO Delete: "api-ms-win-core-string-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.329 INFO Delete: "api-ms-win-core-synch-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.329 INFO Delete: "api-ms-win-core-synch-l1-2-0.dll" (attempt 1)
Jun 18 12:03:43.329 INFO Delete: "api-ms-win-core-sysinfo-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.329 INFO Delete: "api-ms-win-core-timezone-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.334 INFO Delete: "api-ms-win-core-util-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.334 INFO Delete: "api-ms-win-crt-conio-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.334 INFO Delete: "api-ms-win-crt-convert-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.334 INFO Delete: "api-ms-win-crt-environment-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.334 INFO Delete: "api-ms-win-crt-filesystem-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.334 INFO Delete: "api-ms-win-crt-heap-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.339 INFO Delete: "api-ms-win-crt-locale-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.339 INFO Delete: "api-ms-win-crt-math-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.339 INFO Delete: "api-ms-win-crt-multibyte-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.339 INFO Delete: "api-ms-win-crt-private-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.344 INFO Delete: "api-ms-win-crt-process-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.344 INFO Delete: "api-ms-win-crt-runtime-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.344 INFO Delete: "api-ms-win-crt-stdio-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.344 INFO Delete: "api-ms-win-crt-string-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.344 INFO Delete: "api-ms-win-crt-time-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.344 INFO Delete: "api-ms-win-crt-utility-l1-1-0.dll" (attempt 1)
Jun 18 12:03:43.349 INFO Delete: "bin" (attempt 1)
Jun 18 12:03:43.349 INFO Delete: "blink_image_resources_200_percent.pak" (attempt 1)
Jun 18 12:03:43.349 INFO Delete: "Code.exe" (attempt 1)
Jun 18 12:03:43.354 INFO Delete: "Code.VisualElementsManifest.xml" (attempt 1)
Jun 18 12:03:43.354 INFO Delete: "CodeSignSummary-ca435f23-0cec-45ec-b54b-27be2e2f3403.md" (attempt 1)
Jun 18 12:03:43.359 INFO Delete: "content_resources_200_percent.pak" (attempt 1)
Jun 18 12:03:43.359 INFO Delete: "content_shell.pak" (attempt 1)
Jun 18 12:03:43.389 INFO Delete: "d3dcompiler_47.dll" (attempt 1)
Jun 18 12:03:43.389 INFO Delete: "ffmpeg.dll" (attempt 1)
Jun 18 12:03:43.389 INFO Delete: "icudtl.dat" (attempt 1)
Jun 18 12:03:43.389 INFO Delete: "libEGL.dll" (attempt 1)
Jun 18 12:03:43.429 INFO Delete: "libGLESv2.dll" (attempt 1)
Jun 18 12:03:43.429 INFO Delete: "locales" (attempt 1)
Jun 18 12:03:43.514 INFO Delete: "msvcp140.dll" (attempt 1)
Jun 18 12:03:43.514 INFO Delete: "natives_blob.bin" (attempt 1)
Jun 18 12:03:43.519 INFO Delete: "node.dll" (attempt 1)
Jun 18 12:03:43.519 INFO Delete: "pdf_viewer_resources.pak" (attempt 1)
Jun 18 12:03:43.519 INFO Delete: "resources" (attempt 1)
Jun 18 12:04:15.569 INFO Delete: "snapshot_blob.bin" (attempt 1)
Jun 18 12:04:15.571 INFO Delete: "ucrtbase.dll" (attempt 1)
Jun 18 12:04:15.572 INFO Delete: "ui_resources_200_percent.pak" (attempt 1)
Jun 18 12:04:15.574 INFO Delete: "vcruntime140.dll" (attempt 1)
Jun 18 12:04:15.575 INFO Delete: "views_resources_200_percent.pak" (attempt 1)
Jun 18 12:04:15.641 INFO Rename: "api-ms-win-core-console-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.717 INFO Rename: "api-ms-win-core-datetime-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.720 INFO Rename: "api-ms-win-core-debug-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.722 INFO Rename: "api-ms-win-core-errorhandling-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.723 INFO Rename: "api-ms-win-core-file-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.724 INFO Rename: "api-ms-win-core-file-l1-2-0.dll" (attempt 1)
Jun 18 12:04:15.725 INFO Rename: "api-ms-win-core-file-l2-1-0.dll" (attempt 1)
Jun 18 12:04:15.726 INFO Rename: "api-ms-win-core-handle-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.728 INFO Rename: "api-ms-win-core-heap-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.729 INFO Rename: "api-ms-win-core-interlocked-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.730 INFO Rename: "api-ms-win-core-libraryloader-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.732 INFO Rename: "api-ms-win-core-localization-l1-2-0.dll" (attempt 1)
Jun 18 12:04:15.734 INFO Rename: "api-ms-win-core-memory-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.735 INFO Rename: "api-ms-win-core-namedpipe-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.737 INFO Rename: "api-ms-win-core-processenvironment-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.738 INFO Rename: "api-ms-win-core-processthreads-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.739 INFO Rename: "api-ms-win-core-processthreads-l1-1-1.dll" (attempt 1)
Jun 18 12:04:15.740 INFO Rename: "api-ms-win-core-profile-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.742 INFO Rename: "api-ms-win-core-rtlsupport-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.744 INFO Rename: "api-ms-win-core-string-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.746 INFO Rename: "api-ms-win-core-synch-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.747 INFO Rename: "api-ms-win-core-synch-l1-2-0.dll" (attempt 1)
Jun 18 12:04:15.750 INFO Rename: "api-ms-win-core-sysinfo-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.751 INFO Rename: "api-ms-win-core-timezone-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.752 INFO Rename: "api-ms-win-core-util-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.754 INFO Rename: "api-ms-win-crt-conio-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.754 INFO Rename: "api-ms-win-crt-convert-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.755 INFO Rename: "api-ms-win-crt-environment-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.756 INFO Rename: "api-ms-win-crt-filesystem-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.757 INFO Rename: "api-ms-win-crt-heap-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.758 INFO Rename: "api-ms-win-crt-locale-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.759 INFO Rename: "api-ms-win-crt-math-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.760 INFO Rename: "api-ms-win-crt-multibyte-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.761 INFO Rename: "api-ms-win-crt-private-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.761 INFO Rename: "api-ms-win-crt-process-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.762 INFO Rename: "api-ms-win-crt-runtime-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.763 INFO Rename: "api-ms-win-crt-stdio-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.764 INFO Rename: "api-ms-win-crt-string-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.765 INFO Rename: "api-ms-win-crt-time-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.766 INFO Rename: "api-ms-win-crt-utility-l1-1-0.dll" (attempt 1)
Jun 18 12:04:15.767 INFO Rename: "bin" (attempt 1)
Jun 18 12:04:15.839 INFO Rename: "blink_image_resources_200_percent.pak" (attempt 1)
Jun 18 12:04:15.843 INFO Rename: "Code.exe" (attempt 1)
Jun 18 12:04:15.897 INFO Rename: "Code.exe" (attempt 2)
Jun 18 12:04:16.099 INFO Rename: "Code.exe" (attempt 3)
Jun 18 12:04:16.551 INFO Rename: "Code.exe" (attempt 4)
Jun 18 12:04:17.352 INFO Rename: "Code.exe" (attempt 5)
Jun 18 12:04:18.604 INFO Rename: "Code.exe" (attempt 6)
Jun 18 12:04:20.582 INFO Rename: "Code.exe" (attempt 7)
Jun 18 12:04:23.061 INFO Rename: "Code.exe" (attempt 8)
Jun 18 12:04:26.310 INFO Rename: "Code.exe" (attempt 9)
Jun 18 12:04:30.404 INFO Rename: "Code.exe" (attempt 10)
Jun 18 12:04:35.407 INFO Rename: "Code.exe" (attempt 11)
Jun 18 12:04:35.456 ERRO Access is denied. (os error 5)
```

**Comment by YourAverageCoderr** [comment]
Update: reinstalling VSCode from grabbing the latest installer keeps screaming at me for access denied errors. I can't delete C:\Program Files\Microsoft VS Code\Code.exe, no matter what I try. This keeps happening with Chromium applications I'm pretty sure, as this happened with an Electron app I made a while back and I couldn't delete electron.exe (still can't), as well as with SRWare Iron when I tried to uninstall it.
